packages/preview/zju-bubble/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "zju-bubble"
```

Comment:
This is a little pedantic, but the [naming rules](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules) state that the non-descriptive part should precede the organization name, so this would have to be `bubble-zju`